src/neuroconv/datainterfaces/ecephys/ced/ceddatainterface.py

```diff
@@ -28,11 +36,17 @@ def get_all_channels_info(cls, file_path: FilePathType):
         """Retrieve and inspect necessary channel information prior to initialization."""
         return cls.RX.get_all_channels_info(file_path=file_path)
 
+    def __new__(cls, *args, **kwargs):
+        getattr(cls, "RX")
```

Comment:
While issuing a manual call like the one in the PR description works fine with the metaclass override, in order for a fresh instance of the class to automatically import and set the class, the call must be triggered at this level.